css/style.css

```diff
@@ -0,0 +1,87 @@
+body {
+  background: #1c1a19;
+}
+
+a {
+  text-decoration: none;
+}
+
+li {
+  list-style: none;
+}
+
+.hamburger {
+  padding-right: 30px;
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 3px;
+  margin: 5px auto;
+  background-color: white;
+}
+
+.topnav {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  font-family: 'Roboto', sans-serif;
+  width: 100%;
+  position: fixed;
+  background: #3c3a39;
+}
```

Comment:
- [ ] Please increase the height of the navbar and add more padding left and right to match the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/170719514-50129d99-ea2e-4f55-a9df-e64c1dbd46f6.png)  |![image](https://user-images.githubusercontent.com/68796232/170719489-e0e581de-31ed-4891-b8b6-fee41210767a.png) |